tests/unit/test_exceptions.py

```diff
@@ -87,3 +93,28 @@ def testInstantiation(self):
             self.assertIsInstance(message, basestring)
             self.assertGreater(len(message), 0)
             self.assertEqual(instance.getErrorCode(), class_.getErrorCode())
+
+    def testInstantiationManual(self):
```

Comment:
We should make this more specific: `testValidationFailureExceptions` seems good.